Trippy/Utils/ParseUtils.h

```diff
@@ -0,0 +1,27 @@
+//
+//  ParseUtils.h
+//  Trippy
+//
+//  Created by Catherine Lu on 7/7/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "Parse/Parse.h"
+#import "Collection.h"
+#import "Location.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@interface ParseUtils : NSObject
++ (NSArray *)getCollectionKeys;
++ (NSArray *)getLocationKeys;
++ (NSString *)getLoggedInUserId;
++ (void) collectionFromPFObj:(PFObject *)obj completion:(void (^)(Collection *collection, NSError *))completion;
++ (Location *)locationFromPFObj:(PFObject *)obj;
++ (PFObject *)newPFObjFromCollection:(Collection *)collection;
++ (PFObject *)newPFObjFromLocation:(Location *)loc;
++ (PFObject *)oldPFObjFromLocation:(Location *)loc;
```

Comment:
I would recommend changing the name, maybe simply dropping the "old". However, given my suggestion at the top of the file, maybe this method doesn't need to exist? Or else, should just return an ID.